extensions/wikia/AdEngine/js/slot/scrollHandler.js

```diff
@@ -1,67 +1,110 @@
 /*global define*/
 define('ext.wikia.adEngine.slot.scrollHandler', [
-    'ext.wikia.adEngine.adContext',
-    'ext.wikia.adEngine.adHelper',
-    'wikia.log',
-    'wikia.document',
-    'wikia.window'
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adHelper',
+	'wikia.log',
+	'wikia.document',
+	'wikia.window'
 ], function (adContext, adHelper, log, doc, win) {
-    'use strict';
+	'use strict';
+	var logGroup = 'ext.wikia.adEngine.slot.scrollHandler',
+		isRefreshed = {},
+		reloadedView = {},
+		context = adContext.getContext(),
+		config = context.opts.scrollHandlerConfig || {
+			oasis: {
+				PREFOOTER_LEFT_BOXAD: {
+					reloadedViewMax: 1
+				},
+				PREFOOTER_RIGHT_BOXAD: {
+					reloadedViewMax: 3
+				}
+			},
+			mercury: {
+				MOBILE_PREFOOTER: {
+					reloadedViewMax: 1
+				}
+			}
+		},
+		skin = context.targeting.skin === 'oasis' ? 'oasis' : 'mercury';
```

Comment:
Do we really want to set `skin` to `'mercury'` each time `context.targeting.skin` isn't `'oasis'`? What does `context.targeting.skin` return for Mercury? Can't we just go with `skin = context.targeting.skin`? :japanese_ogre: